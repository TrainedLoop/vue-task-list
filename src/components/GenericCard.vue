<template>
    <div class="card">
        <slot></slot>
    </div>
</template>
<style scoped lang="scss">
.card {
    border-radius: .5rem;
    padding: 1.5rem;
    box-shadow: $shadow;
    background-color: $color-neutral-light-2;
    width: fit-content;
}
</style>