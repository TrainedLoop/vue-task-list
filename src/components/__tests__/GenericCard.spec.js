import { shallowMount } from '@vue/test-utils';
import GenericCard from '../GenericCard.vue';

describe('GenericInput.vue Tests', () => {
  it('renders component correctly', () => {
    const wrapper = shallowMount(GenericCard);
    expect(wrapper.element).toMatchSnapshot();
  });
});
