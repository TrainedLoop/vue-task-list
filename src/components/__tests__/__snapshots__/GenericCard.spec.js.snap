// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenericInput.vue Tests renders component correctly 1`] = `
<div
  class="card"
>
  
  
</div>
`;
